XnConvert.yaml

```diff
@@ -0,0 +1,80 @@
+name: XnConvert
+buildsystem: simple
+sources:
+  - type: archive
+    url: https://download.xnview.com/old_versions/XnConvert-179-linux-x64.tgz
```

Comment:
It should use extra-data instead, as we're now indirectly violating their license by using test builds.

Reply:
Is this an old build, or just a stable URL to the current one? I don't see what in the license prohibits this, it's very clear:

```You are granted the right to use and to make an unlimited number of copies of this software.```

Reply:
Yeah, i think avoiding extra data when not needed due to licensing issues is a much better experience. We *do* have to be very careful about documenting licenses and stuff though, both in appdata and in the manifest.

Reply:
> Is this an old build, or just a stable URL to the current one?

There was an update on March 4. The current version is 1.80, but we have a URL for this as well:
https://download.xnview.com/old_versions/XnConvert-180-linux-x64.tgz

Reply:
@pierrexn 

Can I assume you're okay with this?